scala/scala.bzl

```diff
@@ -162,9 +162,28 @@ def _compile(ctx, cjars, dep_srcjars, buildijar):
     all_srcjars = set(srcjars + list(dep_srcjars))
     # look for any plugins:
     plugins = _collect_plugin_paths(ctx.attr.plugins)
+    dependency_analyzer_plugin_jars = []
+    dependency_analyzer_mode = "off"
+    compiler_classpath_jars = cjars
+
+    # "off" mode is used as a feature toggle, that preserves original behaviour
+    if (hasattr(ctx.attr, 'dependency_analyzer_mode')):
+      if (ctx.attr.dependency_analyzer_mode not in ["error", "warn", "off"]):
+        fail("Incorrect mode of dependency analyzer plugin! Mode must be 'error', 'warn' or 'off'")
+      elif (ctx.attr.dependency_analyzer_mode != "off"):
+        dependency_analyzer_mode = ctx.attr.dependency_analyzer_mode
+        dep_plugin = ctx.attr._dependency_analyzer_plugin
+        plugins += [f.path for f in dep_plugin.files]
+        dependency_analyzer_plugin_jars = ctx.files._dependency_analyzer_plugin
```

Comment:
No. dependency_analyzer_plugin_jars needs to be a **list** of jars paths, and dep_plugin.files is a **depset** of jars paths...